<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateTable</class>
 <widget class="QDialog" name="CreateTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>533</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>863</width>
    <height>533</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>863</width>
    <height>533</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lbNewTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Table Name: Table Name</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblCreateTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>821</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="txbrCreateTableSql">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>821</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnAddRow">
      <property name="text">
       <string>Add New Row</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDeleteRow">
      <property name="text">
       <string>Delete Last Row</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCreateTableGenerateSql">
      <property name="text">
       <string>Generate SQL</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="btnExecute">
      <property name="text">
       <string>Execute</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCopyToteSqlInput">
      <property name="text">
       <string>Copy To Query Edit Area</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
